programs/x509/crl_app.c

```diff
@@ -72,6 +72,14 @@ int main( int argc, char *argv[] )
     mbedtls_x509_crl crl;
     int i;
     char *p, *q;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 113, are displaying it without the negative. We should be self-consistent within the same program.